torch/_inductor/select_algorithm.py

```diff
@@ -1287,7 +1578,7 @@ def make_kernel_render(out_node):
             },
             mutated_inputs=mutated_inputs,
             workspace_arg=workspace_arg,
-            allowed_prologue_inps=kernel.prologue_supported_inputs.copy(),
+            allowed_prologue_inps=prologue_supported_inputs,
```

Comment:
Can we keep the clone for now ? i had some concern around sharing mutable state..

Reply:
oh its done above..